libs/src/api/user.js

```diff
@@ -61,9 +61,14 @@ class Users extends Base {
    * @example
    * getFollowIntersectionUsers(100, 0, 1, 1) - IDs must be valid
    */
-  async getFollowIntersectionUsers (limit = 100, offset = 0, followeeUserId, followerUserId) {
-    this.REQUIRES(Services.DISCOVERY_PROVIDER)
-    return this.discoveryProvider.getFollowIntersectionUsers(limit, offset, followeeUserId, followerUserId)
+  async getFollowIntersectionUsers (limit = 100, offset = 0, followeeUserId) {
+    const account = this.userStateManager.getCurrentUser()
+    if (account) {
+      const followerUserId = account.user_id
+      this.REQUIRES(Services.DISCOVERY_PROVIDER)
```

Comment:
let's keep the this.REQUIRES first (here and elsewhere), as it will throw immediately and we wouldn't need to do any more work